ddtrace/contrib/langchain/patch.py

```diff
@@ -590,6 +593,7 @@ def traced_embedding(langchain, pin, func, instance, args, kwargs):
                 for idx, text in enumerate(input_texts):
                     span.set_tag_str("langchain.request.inputs.%d.text" % idx, integration.trunc(text))
             span.set_metric("langchain.request.input_count", len(input_texts))
+            span.set_tag_str("langchain.request.inputs.chunk_size", chunk_size)
```

Comment:
For simplicity, we don't actually need to enhance the APM integration. If you'd like to, I would suggest moving the APM integration change into a separate PR.